build_tools/github_actions/benchmarks/benchmark_test_matrix.py

```diff
@@ -0,0 +1,64 @@
+"""
+Benchmark test matrix definitions.
+
+This module contains the benchmark_matrix dictionary which defines all benchmark tests.
+Benchmark tests run only on nightly CI builds and are merged into test_matrix by configure_ci.py.
+"""
+
+from pathlib import Path
+
+# Note: these paths are relative to the repository root.
+BENCHMARK_SCRIPT_DIR = Path("./build_tools/github_actions/benchmarks/scripts")
```

Comment:
you can actually use something like this:

```suggestion
THIS_SCRIPT_DIR = Path(__file__).resolve().parent
```